src/app/components/side-bar/tag-images/form-generator/form-fields/form-color/form-color.component.ts

```diff
@@ -59,6 +57,10 @@ export class FormColorComponent implements OnInit {
       this.activeGroup,
       this.value.color
     );
-    this.formValue.emit({ id: this.form.id, value: this.value });
+
```

Comment:
this also does not seeming to run for me at all so I'm never seeing the Save Tags button.
for each of the components, i don't see this.geoDataService.setFeatureTag ever being called.  i might be missing something or is there an unpushed commit that you have?